weaviate/collections/classes/config.py

```diff
@@ -1640,6 +1640,7 @@ def _to_dict(self, vectorizer: Optional[Vectorizer] = None) -> Dict[str, Any]:
 
 
 class _ReferencePropertyBase(_ConfigCreateModel):
+    description: Optional[str] = Field(default=None)
```

Comment:
order? Should it be behind name as the less important thing?

Reply:
I've been ordering the attributes of all these classes alphabetically so just defaulted to that here

Is it okay to continue that convention or shall we start switching them to ordering by importance?

Reply:
hmmm, if there is a clear importance order I would have the order reflect that. Name must be included, description will probably rarely be used